docs/advanced/support-new-devices/01_support_new_devices.md

```diff
@@ -134,6 +137,30 @@ Repeat until your device does not produce any more log messages like: `2018-5-1
 
 If none of the existing converters fit you can add custom ones, an external converter example for this can be found [here](https://github.com/Koenkk/zigbee2mqtt.io/blob/master/docs/externalConvertersExample/freepad_ext.js).
 
+### 2.3 Using modern extend
```

Comment:
I'm wondering if this should replace 2.2 completely.

Reply:
Probably, but I will take some time to make a better guide. So I added this as a temporary fix for now.